.pre-commit-config.yaml

```diff
@@ -4,22 +4,22 @@
 #   pre-commit autoupdate
 repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
+    rev: v4.4.0
     hooks:
       - id: trailing-whitespace
       - id: end-of-file-fixer
         exclude: '.*\.pth$'
       - id: debug-statements
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.2.2
+    rev: v3.3.1
     hooks:
       - id: pyupgrade
-        args: [--py36-plus]
+        args: [--py37-plus]
```

Comment:
If dropping 3.6, there's also the classifier and python_requires in setup.py. Plus mention in the PR title.

Reply:
I think also need to update few parts related to Python 3.6 if this PR is going to drop 3.6.

https://github.com/pytest-dev/pytest-cov/blob/master/setup.py#L114

Reply:
Done.

Reply:
Let's also mention the 3.6 drop in the PR title for visibility/transparency.